core/base/v7/inc/ROOT/RDrawable.hxx

```diff
@@ -19,12 +19,14 @@
 #include <memory>
 #include <string>
 
+
 namespace ROOT {
 namespace Experimental {
 
 class RDrawingOptsBase;
 class RMenuItems;
 class RPadBase;
+class RDrawable;
```

Comment:
Why is this needed?

Reply:
Just remaining of original code, where attributes were placed directly into base RDrawable class. Later one can do it again, but probably RDrawable class can be moved into `graf2d/gpadv7` library.